docs/pages/linking/into-your-app.mdx

```diff
@@ -0,0 +1,127 @@
+
+```json app.json
+{
+  "expo": {
+    /* @info <CODE>scheme</CODE> property allows you to add a custom scheme to your app. */
+    "scheme": "myapp"
+    /* @end */
+  }
+}
+```
+
+After adding a custom scheme to your app, you need to [create a new development build](/develop/development-builds/create-a-build/). Once the app is installed on a device, you can open links within your app using `myapp://`.
+
+If the **custom scheme is not defined**, the app will use `android.package` and `ios.bundleIdentifier` as the default schemes in both development and production builds. This is because [Expo Prebuild](/workflow/prebuild/) automatically adds these properties as custom schemes for Android and iOS.
+
+## Test the deep link
+
+You can test a link that opens your app using [`npx uri-scheme`](https://github.com/expo/expo/tree/main/packages/uri-scheme#readme), which is a command-line utility for interacting and testing URI schemes.
+
+For example, if your app has a `/details` screen that you want to open when a user taps on a link (either through another app or a web browser), you can test this behavior by running the following command:
+
+<Terminal
+  cmd={[
+    '# If you have `android.package` or `ios.bundleIdentifier` defined in your app.json',
+    '$ npx uri-scheme open com.example.app://somepath/details --android',
+    '',
+    '# If you have a `scheme` defined in your app.json',
+    '$ npx uri-scheme open myapp://somepath/details --ios',
+  ]}
+/>
+
+Running the above command:
+
+- Opens your app's `/details` screen
+- The `android` or `ios` options specify that the link should be opened on Android or iOS
+- Alternatively, you can open the link by searching for it in the device's native browser. For example, open Safari on iOS and type `myapp://` in the address bar to prompt the device to open your app.
+
+<Collapsible summary="Test a link using Expo Go">
+
+By default, [Expo Go](https://expo.dev/go) uses the `exp://` scheme. If you link to `exp://` without specifying a URL address afterward, it will open the app to the home screen. In development, your app's complete URL looks like `exp://127.0.0.1:8081`.
+
+To open the `/details` screen while testing on Expo Go, you can use `npx uri-scheme`:
+
+<Terminal
+  cmd={['$ npx uri-scheme open exp://127.0.0.1:8081/--/somepath/into/app?hello=world --ios']}
+/>
+
+In Expo Go, `/--/` is added to the URL when a path is specified. This indicates to Expo Go that the substring after it corresponds to the deep link path and is not part of the path to the app itself.
+
+By default, `exp://` is replaced with `http://` when opening a URL in Expo Go. You can also use `exps://` to open `https://` URLs. However, `exps://` does not currently support loading sites with insecure TLS certificates.
+
+</Collapsible>
+
+## Handle URLs
+
+You can observe links that launch your app using the [`Linking.useURL()`](/versions/latest/sdk/linking/#useurl) hook from [`expo-linking`](/versions/latest/sdk/linking/).
```

Comment:
might be worth a call-out to the effect of "you can ignore all this with Expo Router"